src/manual-multiple/web/index.html

```diff
@@ -1,13 +1,11 @@
 <aside class="adverts adverts--legacy adverts--manual adverts--[%Tone%]s adverts--tone-[%Tone%]s" data-link-name="creative | manual multiple | [%TrackingId%]">
     <header class="adverts__header">
         <h1 class="adverts__title">
-            <a class="adverts__logo blink" href="%%CLICK_URL_UNESC%%[%TitleURL%]" data-link-name="title" target="_top">
-                {{#svg}}marque{{/svg}}
-                <!-- guardian logo -->
-                <span class="blink__anchor">[%Title%]</a>
-            </a>
+            <a class="adverts__logo brand_logo" href="%%CLICK_URL_UNESC%%[%TitleURL%]" data-link-name="title" target="_top"></a>
+        </h1>
+        <h1 class="adverts_blurb">
```

Comment:
As above: do we need to wrap this in an `h1`?

Reply:
Nope! Changed...